src/wp-admin/theme-install.php

```diff
@@ -540,58 +498,16 @@
 						</div>
 
 						<# if ( ! data.compatible_wp || ! data.compatible_php ) { #>
-							<div class="notice notice-error notice-alt notice-large"><p>
-								<# if ( ! data.compatible_wp && ! data.compatible_php ) { #>
-									<?php
-									_e( 'This theme does not work with your versions of WordPress and PHP.' );
-									if ( current_user_can( 'update_core' ) && current_user_can( 'update_php' ) ) {
-										printf(
-											/* translators: 1: URL to WordPress Updates screen, 2: URL to Update PHP page. */
-											' ' . __( '<a href="%1$s">Please update WordPress</a>, and then <a href="%2$s">learn more about updating PHP</a>.' ),
-											self_admin_url( 'update-core.php' ),
-											esc_url( wp_get_update_php_url() )
-										);
-										wp_update_php_annotation( '</p><p><em>', '</em>' );
-									} elseif ( current_user_can( 'update_core' ) ) {
-										printf(
-											/* translators: %s: URL to WordPress Updates screen. */
-											' ' . __( '<a href="%s">Please update WordPress</a>.' ),
-											self_admin_url( 'update-core.php' )
-										);
-									} elseif ( current_user_can( 'update_php' ) ) {
-										printf(
-											/* translators: %s: URL to Update PHP page. */
-											' ' . __( '<a href="%s">Learn more about updating PHP</a>.' ),
-											esc_url( wp_get_update_php_url() )
-										);
-										wp_update_php_annotation( '</p><p><em>', '</em>' );
-									}
-									?>
-								<# } else if ( ! data.compatible_wp ) { #>
-									<?php
-									_e( 'This theme does not work with your version of WordPress.' );
-									if ( current_user_can( 'update_core' ) ) {
-										printf(
-											/* translators: %s: URL to WordPress Updates screen. */
-											' ' . __( '<a href="%s">Please update WordPress</a>.' ),
-											self_admin_url( 'update-core.php' )
-										);
-									}
-									?>
-								<# } else if ( ! data.compatible_php ) { #>
-									<?php
-									_e( 'This theme does not work with your version of PHP.' );
-									if ( current_user_can( 'update_php' ) ) {
-										printf(
-											/* translators: %s: URL to Update PHP page. */
-											' ' . __( '<a href="%s">Learn more about updating PHP</a>.' ),
-											esc_url( wp_get_update_php_url() )
-										);
-										wp_update_php_annotation( '</p><p><em>', '</em>' );
-									}
-									?>
-								<# } #>
-							</p></div>
+							<?php
+							$compatibility_template = theme_compatibility_template();
+							echo wp_get_admin_notice(
```

Comment:
Is this used instead of `wp_admin_notice()` because of escaping issues? If so, we might consider adding an inline comment to ensure this doesn't get changed in future without awareness of possible escaping issues.